Lib/idlelib/configdialog.py

```diff
@@ -1853,6 +1853,219 @@ def save_all_changed_extensions(self):
             self.ext_userCfg.Save()
 
 
+class FontPage:
+
+    def __init__(self, parent, tab_pages, highlight_sample):
```

Comment:
At step 2, testing FontPage isolated, passing in Func() when creating an instance of FontPage would work.  But the mock could also be set on the instance after creating it.  At step 3, testing FontPage live, FontPage will be created by ConfigDialog.  highlight_sample will not exist until PageHighlight is called to create and initialize its widgits, first.  At that point, it can be retrieved using tab_pages.  Lets leave this for now.